testing-renamed/tests/hello-world.rs

```diff
@@ -0,0 +1,55 @@
+use std::fmt;
+
+use assert_matches::assert_matches;
+use some_name::Template;
+
+pub(crate) mod some {
+    pub(crate) mod deeply {
+        pub(crate) mod nested {
+            pub(crate) mod path {
+                pub(crate) mod with {
+                    #[allow(clippy::single_component_path_imports)] // false positive
```

Comment:
The lint should probably be silent for any visibility but `pub(self)`?

Reply:
Likely so, yes.